gtars/src/overlap/ailist.rs

```diff
@@ -91,29 +146,32 @@ where
             &self.starts[self.header_list[i]..],
             &self.ends[self.header_list[i]..],
             &self.max_ends[self.header_list[i]..],
-            &self.data_list[self.header_list[i]..],
+            &self.stored_intervals[self.header_list[i]..],
         ));
 
         results_list
     }
+
+    fn find_iter<'a>(
+        &'a self,
+        start: I,
+        stop: I,
+    ) -> Box<dyn Iterator<Item = &'a Interval<I, T>> + 'a> {
+        Box::new(IterFind::new(self, start, stop))
+    }
```

Comment:
For my own learning... `'a` is necessary as we need to annotate the `Interval` reference to say,
> This memory is valid so long as the AiList itself is valid

Reply:
Yep! that's correct. "lives at lest as long as 'a" would be the way I'd read it.